app/scripts/views/settings/emails.js

```diff
@@ -23,6 +23,8 @@ define(function (require, exports, module) {
   const EMAIL_REFRESH_SELECTOR = 'button.settings-button.email-refresh';
   const EMAIL_REFRESH_DELAYMS = 350;
 
+  const EMAIL_ENABLED_REGEX = /.+@mozilla\.com$|.+@restmail\.net$|.+@softvisioninc\.eu$|.+@softvision\.(com|ro)$/;
```

Comment:
Do you reckon it's worth getting this from config so that we can control future changes without touching the source code?